backend/editor/api.py

```diff
@@ -36,7 +36,21 @@
 from .exceptions import GithubBranchExistsError, GithubUploadError
 
 # Data model imports
-from .models import Footer, Header
+from .models import (
+    CreateNodeParameters,
+    EditChildrenParameters,
+    EditChildrenResponse,
+    EditEntryParameters,
+    EditEntryResponse,
+    EditFooterParameters,
+    EditFooterResponse,
+    EditHeaderParameters,
+    EditHeaderResponse,
+    EditSynonymParameters,
+    EditSynonymResponse,
+    ImportFromGithubParameters,
+    ImportFromGithubResponse,
+)
```

Comment:
my opinion: I would rather just import models (`from . import models`) and use the dot notation everywhere. (`models.CreateNodeParameters` for example)

It's more a matter of taste tough. Personally when it does not hinders performances or reading, I prefer to avoid direct imports.